functional-test/src/test/groovy/org/rundeck/util/common/jobs/JobUtils.groovy

```diff
@@ -116,26 +118,38 @@ class JobUtils {
         return jobs[0]
     }
 
+
     static Execution waitForExecutionToBe(
             String state,
             String executionId,
             ObjectMapper mapper,
             RdClient client,
-            int iterationGap,
-            int timeout
+            WaitingTime iterationGap,
```

Comment:
I'd recommend providing default values for the `WaitingTime`s

Reply:
I'm not sure if i want devs to be allowed to use this method without providing an explicit waiting time. I think it would make the tests code more clear to the ones that will be reading that code for the first time. Maybe would be better to just check for nulls?

Reply:
I'm worried devs will pick uneccessary long durations because "not sure exactly, what I need, but very_long_duration seems to work locally". This will result in unnecessary slowdowns. I'd recommend defaulting to something short so that overriding would be an explicit choice. Regardless, this is a `nit`. Your call.

Reply:
As discussed, we'll implement this in a next phase.